pkg/security/secl/rules/ruleset.go

```diff
@@ -284,6 +284,13 @@ func GetRuleEventType(rule *eval.Rule) (eval.EventType, error) {
 	return eventType, nil
 }
 
+func (rs *RuleSet) isActionAvailable(eventType eval.EventType, action *Action) bool {
+	if action.Def.Name() == HashAction && eventType != model.FileOpenEventType.String() {
+		return false
+	}
+	return true
+}
+
```

Comment:
With this change we won't be able to have a hash action on exec rules, should we also have this event type here as well?

Reply:
Good point

Reply:
done